spotbugs/src/main/java/edu/umd/cs/findbugs/detect/BadUseOfReturnValue.java

```diff
@@ -62,8 +62,9 @@ public void sawOpcode(int seen) {
         }
 
         if (seen == Const.INVOKEVIRTUAL && "readLine".equals(getNameConstantOperand())
-                && "()Ljava/lang/String;".equals(getSigConstantOperand()) && getClassConstantOperand().startsWith("java/io")
-                && !"java/io/LineNumberReader".equals(getClassConstantOperand())) {
+                && "()Ljava/lang/String;".equals(getSigConstantOperand()) && 
+                ((getClassConstantOperand().startsWith("java/io") && !"java/io/LineNumberReader".equals(getClassConstantOperand())) ||
+                (getClassConstantOperand().startsWith("javax/imageio/stream")))) {
```

Comment:
Code format in these lines have several problems, run `./gradlew spotlessApply` in your local to fix. https://github.com/spotbugs/spotbugs/runs/4491261146?check_suite_focus=true#step:5:64